python-packages/contract_wrappers/src/zero_ex/contract_wrappers/__init__.py

```diff
@@ -0,0 +1,344 @@
+... 'salt': random.randint(1, 100000000000000000),
+... 'makerFee': 0,
+... 'takerFee': 0,
+... 'makerAssetAmount': 1000000000000000000,
+... 'takerAssetAmount': 500000000000000000000,
+... 'expirationTimeSeconds': 999999999999999999999}
+>>> tx_hash = zero_ex_exchange.cancel_order(
+...     order=example_order_2, tx_params=TxParams(from_=maker))
+
+Once the transaction is mined, we can get the details of
+our cancellation through the exchange wrapper.
+
+>>> cancel_event = zero_ex_exchange.get_cancel_event(tx_hash);
+>>> cancelled_order_hash = cancel_event[0].args.orderHash.hex()
+
+**Batching orders**
+
+The 0x exchange contract can also process multiple orders at
+the same time. Here is an example where the taker fills
+two orders in one transaction.
+
+>>> order_1 = {
+... 'makerAddress': '0x5409ed021d9299bf6814279a6a1411a7e866a631',
+... 'takerAddress': '0x0000000000000000000000000000000000000000',
+... 'senderAddress': '0x0000000000000000000000000000000000000000',
+... 'feeRecipientAddress': '0x0000000000000000000000000000000000000000',
+... 'makerAssetData': bytes.fromhex(
+...     'f47261b0000000000000000000000000'
+...     'c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),
+... 'takerAssetData': bytes.fromhex(
+...     'f47261b0000000000000000000000000'
+...     'c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),
+... 'salt': random.randint(1, 100000000000000000),
+... 'makerFee': 0,
+... 'takerFee': 0,
+... 'makerAssetAmount': 100,
+... 'takerAssetAmount': 100,
+... 'expirationTimeSeconds': 1000000000000000000}
+>>> order_hash_1 = generate_order_hash_hex(
+...     order_1, zero_ex_exchange.address)
+>>> signature_1 = sign_hash(
+...     provider, to_checksum_address(maker), order_hash_1)
+>>> order_2 = {
+... 'makerAddress': '0x5409ed021d9299bf6814279a6a1411a7e866a631',
+... 'takerAddress': '0x0000000000000000000000000000000000000000',
+... 'senderAddress': '0x0000000000000000000000000000000000000000',
+... 'feeRecipientAddress': '0x0000000000000000000000000000000000000000',
+... 'makerAssetData': bytes.fromhex(
+...     'f47261b0000000000000000000000000'
+...     'c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),
+... 'takerAssetData': bytes.fromhex(
+...     'f47261b0000000000000000000000000'
+...     'c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),
+... 'salt': random.randint(1, 100000000000000000),
+... 'makerFee': 0,
+... 'takerFee': 0,
+... 'makerAssetAmount': 200,
+... 'takerAssetAmount': 200,
+... 'expirationTimeSeconds': 2000000000000000000}
+>>> order_hash_2 = generate_order_hash_hex(
+...     order_2, zero_ex_exchange.address)
+>>> signature_2 = sign_hash(
+...     provider, to_checksum_address(maker), order_hash_2)
+
+Fill order_1 and order_2 together:
+
+>>> tx_hash = zero_ex_exchange.batch_fill_orders(
+...     orders=[order_1, order_2],
+...     taker_amounts=[1, 2],
+...     signatures=[signature_1, signature_2],
+...     tx_params=TxParams(from_=taker))
+"""
+
+from .tx_params import TxParams
```

Comment:
Are these 3 import statements needed?  If so, why?

Reply:
Put them here to make the imports more convenient: e.g.:
`from contract_wrappers import Exchange`
instead of
`from contract_wrappers.exchange_wrapper import Exchange`